### DISCUSSION TITLE
Templates upgraded to 3.0

### DISCUSSION THREAD

**Discussion by grantspilsbury** [payloadcms/payload#7627] [General]
Hi,
When do you plan on upgrading the templates to v3? Could be a duplicate of https://github.com/payloadcms/payload/discussions/6792

**Comment by paulpopus** [comment]
Hey, the website template has been ugpraded already (although overdue an update, soon!) to v3

```
npx create-payload-app@beta --use-pnpm -t website
```

The ecommerce template work has been started although put on ice right now as we work through the issue queue and towards stability. It will have a lot more features than before!

**Reply by grantspilsbury** [reply]
Amazing. Thanks. Any update on a saas / subscription template? Potentially done by end of the year?

**Reply by grantspilsbury** [reply]
> Hey, the website template has been ugpraded already (although overdue an update, soon!) to v3
> 
> ```
> npx create-payload-app@beta --use-pnpm -t website
> ```
> 
> The ecommerce template work has been started although put on ice right now as we work through the issue queue and towards stability. It will have a lot more features than before!

Thanks @paulpopus . Do you have a rough ballpark idea when this will get done? 1 week, 1 month, 6 months, 1 year

**Reply by paulpopus** [reply]
Hopefully 1 month, hard to say right now but we're keen to get back on it very soon

**Reply by grantspilsbury** [reply]
> paulpopus

Hi @paulpopus. I hope your week is going swimmingly. Thanks for the work you peeps are doing on Payload. Do you have an update on the v3 e-comm template?

**Reply by paulpopus** [reply]
Well that whole endeavour took far longer than we imagined, there's now a full ecommerce plugin we'll commit to growing and improving and the template uses it, see: https://payloadcms.com/docs/ecommerce/overview#template

**Comment by paulpopus** [comment]
Closing this discussion in light of ecommerce plugin and template being released. We're going to capture discussions going forward in this RFC so we can better manage engagement on this topic: https://github.com/payloadcms/payload/discussions/14239

Thank you everyone!
